docs/dev/protobuf-features.md

```diff
@@ -1,24 +1,50 @@
 # Development with new Hedera protobufs
 
-:bangbang: **Links and branch names in this doc will change before the merge to `master`** :bangbang:
+In this note we summarize the procedure for developing a Services
+feature that requires changing the Hedera protobufs. There are 
+two main topics: (1) setting up for development, and (2) opening 
+PRs when development is complete.
 
-Services developers are the target audience for this document. It summarizes
-the steps to take when developing a feature that changes the Hedera protobufs.
+At least some of these steps will be automated over time, but the 
+process as is shouldn't be overly burdensome as long as the issue
+is reasonably scoped and the protobuf changes are designed carefully
+at the beginning of the issue.
 
-Suppose your feature is for issue number `123`. There are basically five steps to 
-set up development for the feature:
+# Setup
+
+Suppose your feature is for issue number `123`. Follow these steps to set up and 
+iterate development of the feature.
  1. Create a `0123-D-DoThings` branch off `develop` in the [Hedera protobufs](https://github.com/hashgraph/hedera-protobufs) 
     repository; make your changes to the _*.proto_ in this branch.
- 2. Create a `0123-M-DoThings` branch off `0883-M-V12ReleasePrep` in the [Hedera protobufs Java API](https://github.com/hashgraph/hedera-protobuf) 
-    repository; copy the _*.proto_ from your `0123-D-DoThings` branch in the Hedera protobufs repo.
- 3. Suppose the `version` element in the `master` _pom.xml_ above is `0.12.0-SNAPSHOT`. Then in 
-    the `0123-M-DoThings` branch, change this to `0.12.0-issue.123-SNAPSHOT`.
- 4. Make sure you have Sonatype OSS credentials in your _settings.xml_ as described [here](../release-automation.md). Then in 
-    the [Hedera protobufs Java API](https://github.com/hashgraph/hedera-protobuf) repository, run `mvn -Prelease deploy`. 
-    If this succeeds, you should see output like,
-    ``` ...  Uploading to ossrh: https://oss.sonatype.org/content/repositories/snapshots/com/hedera/hashgraph/hedera-protobuf-java-api/0.12.0-issue.123-SNAPSHOT/hedera-protobuf-java-api-0.12.0-issue.123-20210202.205133-4.jar ... ```
- 5. Finally, in your `0123-M-DoThings` branch in the main [Hedera Services](https://github.com/hashgraph/hedera-services) 
-    repository (that is, _this_ repository), :exclamation: **make certain** you have merged the `0883-M-UseProtoArtifact` branch :exclamation:, and then 
-    update the `hapi-proto.version` property to `0.12.0-issue.123-SNAPSHOT`.
-
-At this point you should be able to commit your changes and push to GitHub, and CircleCI flows should run successfully.
+ 2. Create a `0123-M-DoThings` branch off `master` in the [Hedera protobufs Java](https://github.com/hashgraph/hedera-protobufs-java) 
```

Comment:
Just curious, `hedera-protobufs-java` is only used to deploy changes, since we should not have any such functionality in `hedera-protobufs` ?

Reply:
Yep, that's it!